### PR TITLE
SAI Port attributes for preemphasis setting

### PR DESCRIPTION
**Pull request by dgsudharsan**
Defined SAI Port attributes for setting preemphasis and other serdes driver settings

### CODE REVIEW

**Comment by kcudnik** [comment]
ASSERT FAILED (on line 445): SAI_PORT_ATTR_SERDES_PREEMPHASIS: expected default value, but none provided

**Comment by kcudnik** [comment]
allowing to set t o internal: #908

File: inc/saiport.h [hunk 1]
```diff
@@ -1131,6 +1131,45 @@ typedef enum _sai_port_attr_t
...
+    /**
+     * @brief Port serdes control pre-emphasis
+     *
+     * List of port serdes pre-emphasis values. The values are of type sai_u32_list_t
+     * where the count is number lanes in a port and the list specifies list of values
+     * to be applied to each lane.
+     *
+     * @type sai_u32_list_t
+     * @flags CREATE_AND_SET
+     * @default empty
```
Comment (lguohan) [inline inc/saiport.h]: internal

Comment (dgsudharsan) [reply inc/saiport.h]: Done

**Pull request merged**